content/kubermatic/master/_index.en.md

```diff
@@ -9,30 +9,73 @@ date = 2019-04-27T16:06:34+02:00
+
+The Kubernetes Autoscaler in the KKP Cluster automatically scales up/down when one of the following conditions is satisfied:
+1. Some pods fail to run in the cluster due to insufficient resources. OR
+2. There are nodes in the cluster that have been underutilized for an extended period (10 minutes by default) and can place their Pods on other existing nodes.
+
+#### Manage all KKP users directly from a single panel
+The Admin panel allows KKP administrators to manage the global settings that impact all KKP users directly. Here as an administrator, you can do the following:
+
+- Customize the way custom links (example: Twitter, Github, Slack) are displayed in the Kubermatic dashboard
+- Control various cluster related settings that influence cluster creation, management and clean up after deletion
+- Select from a range of filters to find and control existing dynamic data centres or add new ones
+- Define Preset types in a Kubernetes Custom Resource Definition (CRD) allowing the assignment of new credential types to supported providers
+- Enable and configure etcd backups for your clusters through Backup Buckets
 
-### Use Kubernetes
+#### Manage worker nodes via the UI or the CLI
+Worker nodes can be managed via the KKP web dashboard. Once you have installed kubectl, you can also manage them via CLI to automate the creation, deletion and up-grade of nodes.
 
-By providing managed Kubernetes clusters for your infrastructure, KKP makes Kubernetes as easy as it can be. KKP empowers you to take advantage of all the advanced features that Kubernetes has to offer and increases the speed, flexibility and scalability of your cloud deployment workflow.
+#### Monitoring, Logging & Alerting
+When it comes to monitoring, no approach fits all use cases. KKP allows you to adjust things to your needs by enabling certain customizations to enable easy and tactical monitoring. 
+KKP provides two different levels of Monitoring, Logging, and Alerting. 
 
-### Update Kubernetes
+1. The first targets only the management components (master, seed, CRDs) and is independent. This is the Master/Seed Cluster MLA Stack and only the KKP Admins can access this monitoring data.
 
-KKP provides live updates of your Kubernetes cluster without disrupting your daily business. Use all new features of Kubernetes as you increase scale.
+2. The other component is the User Cluster MLA Stack which is a true multi-tenancy solution for all your end-users as well as a comprehensive overview for the KKP Admin. It helps to speed up individual progress but lets the Admin keep an overview of the big picture. It can be configured per seed to match the requirements of the organizational structure All users can access monitoring data of the user clusters under the projects that they are members of.
```

Comment:
I think you're missing a dot.

```suggestion
2. The other component is the User Cluster MLA Stack which is a true multi-tenancy solution for all your end-users as well as a comprehensive overview for the KKP Admin. It helps to speed up individual progress but lets the Admin keep an overview of the big picture. It can be configured per seed to match the requirements of the organizational structure. All users can access monitoring data of the user clusters under the projects that they are members of.
```